manifest_merging/utils.py

```diff
@@ -0,0 +1,215 @@
+import csv
+import random
+import requests
+from datetime import datetime
+import string
+import logging
+import boto3
+from botocore.exceptions import ClientError
+
+
+def randomString(stringLength=10):
+    """Generate a random string of fixed length """
+
+    letters = string.ascii_lowercase
+    return "".join(random.choice(letters) for i in range(stringLength))
+
+
+def upload_file(
+    file_name,
+    bucket,
+    object_name=None,
+    aws_access_key_id=None,
+    aws_secret_access_key=None,
+):
+    """Upload a file to an S3 bucket
+
+    :param file_name: File to upload
+    :param bucket: Bucket to upload to
+    :param object_name: S3 object name. If not specified then file_name is used
+    :aws_access_key_id: string
+    :aws_secret_access_key: string
+    :return: True if file was uploaded, else False
+    """
+    # If S3 object_name was not specified, use file_name
+    if object_name is None:
+        object_name = file_name
+
+    # Upload the file
+    if aws_access_key_id and aws_secret_access_key:
+        s3_client = boto3.client(
+            "s3",
+            aws_access_key_id=aws_access_key_id,
+            aws_secret_access_key=aws_secret_access_key,
+        )
+    else:
+        s3_client = boto3.client("s3")
+
+    try:
+        msg = f"upload_file {file_name} in {bucket}, object: {object_name}"
+        logging.info(msg)
+        s3_client.upload_file(file_name, bucket, object_name)
+    except ClientError as e:
+        logging.error(e)
+        return False
+    return True
+
+
+def create_presigned_url(
+    bucket_name,
+    object_name,
+    aws_access_key_id=None,
+    aws_secret_access_key=None,
+    expiration=3600,
+):
+    """Generate a presigned URL to share an S3 object
+
+    :param bucket_name: string
```

Comment:
I know it was my bad but can you help to reformat the docstring so that it is consistent with other functions? Thanks!